core/src/main/java/org/infinispan/configuration/global/GlobalMetricsConfiguration.java

```diff
@@ -13,7 +13,9 @@ public class GlobalMetricsConfiguration {
 
    public static final AttributeDefinition<Boolean> GAUGES = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.GAUGES, true).immutable().build();
    public static final AttributeDefinition<Boolean> HISTOGRAMS = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.HISTOGRAMS, false).immutable().build();
+   @Deprecated(forRemoval = true, since = "16.0")
    public static final AttributeDefinition<String> PREFIX = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.PREFIX, "").immutable().build();
+   @Deprecated(forRemoval = true, since = "16.0")
    public static final AttributeDefinition<Boolean> NAMES_AS_TAGS = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.NAMES_AS_TAGS, false).immutable().build();
    public static final AttributeDefinition<Boolean> ACCURATE_SIZE = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.ACCURATE_SIZE, false).build();
    public static final AttributeDefinition<Boolean> JVM = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.JVM, true).build();
```

Comment:
Do you want to add the `@Deprecated` annotation to the `LEGACY` attribute and related methods (below, line 22)?

Reply:
Done